### PR TITLE
Simplify install command

### PR DESCRIPTION
**Pull request by seglo**

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 1]
```diff
@@ -156,11 +156,10 @@ helm repo update
 
 [source,shell script]
 ----
-helm install akka-operator akka-operator-helm/charts/akka-operator-<version>.tgz \
-  --namespace lightbend  \
-  --set image.repository=803424716218.dkr.ecr.eu-central-1.amazonaws.com/akka-operator  \
-  --set image.tag=<tag>  \
-  --set serviceAccount.name=akka-operator
+helm install akka-operator akka-operator-helm/akka-operator \
+  --version=<version> \
```
Comment (seglo) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:160]: Should we hardcode version for now until we setup a var substitution?

I just released `0.4.0`.

Comment (andreaTP) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:160]: I don't know much about those docs but you should be able to use something like:
https://github.com/lightbend/cloudflow/blob/master/docs/docs-source/docs/antora.yml#L8

var substitution is built-in in ascii-doc/antora

**Review by andreaTP — APPROVED** [review]

File: docs-source/docs/modules/deployment/pages/aws-install.adoc (hunk 1 above)
Comment (seglo) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:160]: Yes, we should do that. I meant we should automate the docs update as part of a release, but that will need to wait until this wip branch is merged I guess.

**Review by octonato — APPROVED** [review]
LGTM

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 2]
```diff
@@ -205,9 +204,8 @@ helm repo update
 
 [source,shell script]
 ----
-helm upgrade akka-operator akka-operator-helm/charts/akka-operator-<version>.tgz \
+helm upgrade akka-operator akka-operator-helm/akka-operator \
   --namespace lightbend \
-  --set image.repository=803424716218.dkr.ecr.eu-central-1.amazonaws.com/akka-operator \
   --set image.tag=<version>
```
Comment (patriknw) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:209]: strange that this is using `image.tag` but helm install is using version

**Pull request merged**